docs/editors/setup.md

```diff
@@ -153,7 +153,7 @@ extension for [coc.nvim](https://github.com/neoclide/coc.nvim).
 " Linter
 let g:ale_linters = { "python": ["ruff"] }
 " Formatter
-let g:ale_fixers = { "python": ["ruff_format"] }
+let g:ale_fixers = { "python": ["ruff", "ruff_format"] }
```

Comment:
Let's update the comment to state that `ruff` is for fixing all auto-fixable problems while `ruff_format` runs the formatter

Reply:
Additional information added.